cli/azd/pkg/infra/provisioning/bicep/bicep_provider.go

```diff
@@ -51,6 +51,9 @@ type BicepDeploymentDetails struct {
 	Parameters azure.ArmParameters
 	// TemplateOutputs are the outputs as specified by the template.
 	TemplateOutputs azure.ArmTemplateOutputs
+	// Target is the resource in azure that represents the deployment that will happen. This includes the scope as well as
+	// the name of the deployment.
```

Comment:
With this being an exported type, and I wonder if the reader might understand the full context here.

```suggestion
	// Target is the unique resource in azure that represents the deployment that will happen. A target can be scoped to either subscriptions, or resource groups.
```